l10n_it_withholding_tax/models/account.py

```diff
@@ -527,24 +527,34 @@ def _compute_payments_widget_reconciled_info(self):
                         payment_val["wt_move_line"] = False
         return
 
-    def action_register_payment(self):
-        """
-        Set net to pay how default amount to pay
+    def _wt_unlink_statements_move_states(self):
```

Comment:
Non vedo queste modifiche in https://github.com/OCA/l10n-italy/pull/3534, da dove vengono?

Reply:
https://github.com/OCA/l10n-italy/pull/3289

Reply:
visto ora che facendo cherry-pick si è portato questo modifiche